packages/@okta/vuepress-site/docs/guides/configure-ciba/main/index.md

```diff
@@ -172,7 +172,7 @@ You can also use the [Apps API](https://developer.okta.com/docs/api/openapi/okta
 You can use either the org authorization server or a custom authorization server with CIBA. In this example, use the `default` custom authorization server.
 
 1. In the Admin Console, go to **Security** > **API**.
-1. On the **Authorization Servers** tab, select the edit icon to the right of the authorization server that you want to use. In this example, use the `default` custom authorization server.
+1. On the **Authorization Servers** tab, select the authorization server that you want to use and click edit. In this example, use the `default` custom authorization server.
```

Comment:
```suggestion
1. On the **Authorization Servers** tab, select the authorization server that you want to use and click the edit icon. In this example, use the `default` custom authorization server.
```

Reply:
This was a sentence I needed to update to get to 80. The edit icon is the familiar pencil icon. I thought it was sufficient to simply say edit in this case.